src/cascadia/TerminalApp/TerminalPage.xaml

```diff
@@ -98,6 +98,7 @@ the MIT License. See LICENSE in the project root for license information. -->
             VerticalAlignment="Stretch" />
 
         <mux:InfoBar x:Name="KeyboardWarningInfoBar"
+                     x:Load="False"
```

Comment:
You do unfortunately need to use the `FindName` trick with Load; as written it will crash i think

Reply:
(this is because the member isn't populated until the control is loaded, but FindName _loads it_ (instead of finding it. official API!)

Reply:
_This_ is really good to know. Just hit this in my own implementation haha.